cfpb/cfgov-refresh
# Assume utf8 for file names in python2/3 for credit card agreements import

## Changes

- Decodes paths from the zipfile as utf8 always (instead of utf8 in python3 and cp1252 in python2)

## Testing

1. Download the file at https://files.consumerfinance.gov/a/assets/Credit_Card_Agreements_2019_Q1.zip (beware, nearly a gigabyte)
2. Run `./cfgov/manage.py import_agreements --path "Credit_Card_Agreements_2019_Q1.zip"` (with the path to your downloaded file
3. There should be no unicode errors thrown
